scripts/build.sh

```diff
@@ -50,6 +50,10 @@ case "${1:-}" in
 make)
     cmake "$WORKSPACE"
     make -j
+    cp "$WORKSPACE/src/LICENSE" output/lpac-LICENSE
+    cp "$WORKSPACE/euicc/LICENSE" output/libeuicc-LICENSE
```

Comment:
Do you forget to upload LICENSE files for /src and /euicc?

Reply:
It's already in the project, it's always been there.